docs/open-roles/cloud-service-certification-project-maintainer.md

```diff
@@ -0,0 +1,36 @@
+# CSC Project Maintainer - Cloud Service Certification 
+Helping run [Cloud Service Certification](https://github.com/finos/cloud-service-certification) with the Cloud Service Certification Lead Maintainer as CSC Project Maintainer is a fun open source role involving a broad set of tasks that need to be taken care of, including ...
+
+## Role Requirements
+
+- You're an experienced senior level cloud services engineer within the financial services industry and understand the relationship between cloud engineering, finacial services regulation, cyber security, risk and banking policies within an enterprise banking setting.
```

Comment:
word `financial services regulation` slightly differently, to `ideally with`.

Reply:
@peter-thomas-db

This is addressed in the commit below below ...

- update engineering experience to ideally within FS industry 5f5c4d1

James.